NEWS

```diff
@@ -77,7 +77,8 @@
     ZMQ_DGRAM
   All these sockets are THREAD SAFE, unlike the existing socket types. They do
   NOT support multipart messages (ZMQ_SNDMORE/ZMQ_RCVMORE).
-  All these sockets minus ZMQ_SERVER and ZMQ_CLIENT also support UDP as transport.
+  All these sockets minus ZMQ_SERVER and ZMQ_CLIENT also support UDP as transport,
```

Comment:
Agh sorry! For some reasons I was convinced gather and scatter supported it too.
https://github.com/zeromq/libzmq/pull/2192